### ISSUE TITLE
add php support

### ISSUE THREAD

**Issue by detain** [blarApp/blarify#158]
could you add php support?

**Comment by berrazuriz1** [comment]
Hi! this is not in our short term road map, but would love the help. If you like we can meet and I can show how you could add php to our supported languages

**Comment by LZING** [comment]
> Hi! this is not in our short term road map, but would love the help. If you like we can meet and I can show how you could add php to our supported languages

Is it possible to support other languages ​​by modifying the configuration of treesitter?

**Comment by berrazuriz1** [comment]
Yes, but the ideal is add another language parser that inherits from the BaseParser, this way we maintain a standard process of creating the graph from diferents languages
